Src/Newtonsoft.Json/FormatterAssemblyStyle.cs

```diff
@@ -9,12 +9,12 @@ namespace System.Runtime.Serialization.Formatters
     public enum FormatterAssemblyStyle
     {
         /// <summary>
-        /// In simple mode, the assembly used during deserialization need not match exactly the assembly used during serialization. Specifically, the version numbers need not match as the <c>LoadWithPartialName</c> method is used to load the assembly.
+        /// In simple mode, the assembly used during deserialization need not match exactly the assembly used during serialization. Specifically, the version numbers need not match as the <see cref="M:System.Reflection.Assembly.LoadWithPartialName(String)"/> method is used to load the assembly.
```

Comment:
Member ID syntax is also necessary here. The referenced method is not actually present in all targeted platforms. Perhaps a target-specific `#if` block is needed here to have target-specific comments?